ansible/roles/fanout/library/port_config_gen.py

```diff
@@ -150,10 +150,13 @@ def _read_from_port_config(filepath):
         port_config = {}
         with open(filepath) as fd:
             lines = fd.readlines()
-            header = lines[0].strip("#\n ")
+            header_index = 0
+            while "alias" not in lines[header_index]:
```

Comment:
What if "alias" in the comment line? Are comment lines always before the header line? If so, we can use last comment line as header

Reply:
updated